.gitignore

```diff
@@ -13,3 +13,8 @@ project/project/
 yarn.lock
 package-lock.json
 digiroad2.boot.log
+/build/
+.classpath
+.project
+.settings/
+WebContent/
```

Comment:
Is this suppose to be merged? Just checking.

Reply:
Yes these changes enable to use eclipse. I use eclipse for git stashes

Reply:
Nothing against Eclipse, it was my first ever IDE, but you should really give it a try to IntelliJ IDE! 😄